### DISCUSSION TITLE
Corporate sponsorship

### DISCUSSION THREAD

**Discussion by jstaffans-relex** [tomkerkhove/promitor#2213] [General]
We would potentially be interested in sponsoring Tom or another contributor to get certain features implemented in Promitor, e.g. https://github.com/tomkerkhove/promitor/issues/1665. This would need to be a handled as a freelance contract as sponsoring through Github is not possible for accounting reasons. 

How could we proceed?

**Comment by tomkerkhove** [comment]
Unfortunately, I cannot do freelancing work as I work for Microsoft but I appreciate the offer though!

That being said, I understand regular sponsorship is not possible (and not a must) but I do welcome contributions if my changes are too slow.

> PS: Always happy to enlist end-users on https://promitor.io/.
